_articles/metrics.md

```diff
@@ -73,6 +74,10 @@ For example, if your project lands on the front page of Hacker News, you'll prob
 
 Try to figure out where your audience is coming from and ask others for feedback on your project page to figure out which of these two issues you're facing.
 
+Once you know that people are using your project, you might want to try to figure out what they are doing with it.  Are they building on it by forking your code and adding features?  Are they using it for science or business?
+
+In order to determine this, you can use tools like [depsy](http://depsy.org), or you can [publish your software and make it citable](https://guides.github.com/activities/citable-code/).  Then if you also ask users to cite your code, particularly in academia, you can use existing citation tools (such as [Google Scholar](https://scholar.google.com)) to understand what users say they are doing with it.
```

Comment:
I like the general focus of this contribution (figure out what your project is being used for), but agree that citations are too sector-specific.

If you remove the 2nd paragraph with specific tools I'd be :thumbsup:

Ideally, we could think of some more generally applicable examples (ex. ask people to submit examples or testimonials, or search on Libraries.io, search for mentions on Twitter, etc), but I'm fine with just generalities too.